shared/components/Conference/index.js

```diff
@@ -1,27 +1,18 @@
 import React from 'react';
-import SiteFooter from '../SiteFooter';
-import Hero from '../Hero';
-import RedBadgerBanner from '../RedBadgerBanner';
-import MailingList from '../MailingList';
-import ConferenceAbout from '../ConferenceAbout';
-import NextConferenceEvent from '../NextConferenceEvent';
-import ShareYourIdeas from '../ShareYourIdeas';
+// import SiteFooter from '../SiteFooter';
```

Comment:
Note that these are commented out to make your review easier as we don't actually use this component yet.